collectors/health.go

```diff
@@ -769,6 +802,103 @@ func (c *ClusterHealthCollector) collect() error {
 	c.RemappedPGs.Set(stats.OSDMap.OSDMap.NumRemappedPGs)
 	c.TotalPGs.Set(stats.PGMap.NumPGs)
 
+	cmd = c.cephHealthDetailCommand()
+	buf, _, err = c.conn.MonCommand(cmd)
+	if err != nil {
+		return err
+	}
+
+	hdstats := &cephHealthDetailStats{}
+	if err := json.Unmarshal(buf, hdstats); err != nil {
+		return err
+	}
+
+	var (
+		slowOpsBlockedRegex         = regexp.MustCompile(`([\d]+) ops are blocked > ([\d\.]+) sec`)
+		slowRequestSingleOSDRegex   = regexp.MustCompile(`osd.([\d]+) has blocked requests > ([\d\.]+) sec`)
+		slowRequestMultipleOSDRegex = regexp.MustCompile(`osds ([\d,]+) have blocked requests > ([\d\.]+) sec`)
+
+		secToOpsBlocked     = make(map[float64]int)
+		osdToSecondsBlocked = make(map[int]float64)
+	)
+
+	for key, check := range hdstats.Checks {
+		if key == "REQUEST_SLOW" {
+			for _, detail := range check.Details {
+				matched := slowOpsBlockedRegex.FindStringSubmatch(detail.Message)
+				if len(matched) == 3 {
+					v, err := strconv.Atoi(matched[1])
+					if err != nil {
+						return err
+					}
+
+					f, err := strconv.ParseFloat(matched[2], 64)
+					if err != nil {
+						return err
+					}
+
+					secToOpsBlocked[f] = v
```

Comment:
Indeed. We can certainly continue to avoid parsing other things. Updated.

I'd lean towards failing loudly on such a failure because under no known circumstances should the parsing fail. That usually means something big has changed in Ceph (since not unlikely) and we should take a look ASAP.